src/state/user/reducer.ts

```diff
@@ -112,6 +112,9 @@ const userSlice = createSlice({
       state.timestamp = currentTimestamp()
     },
     addSerializedPair(state, { payload: { serializedPair } }) {
+      if (!state.pairs) {
```

Comment:
i don't know, but `tokens` has a similar check which is why i added it. it's probably fine to not to add for `pairs` now

Reply:
i think this is relevant for another comment on a PR but there are tools to migrate redux state - i just think if our typechecker is lying to us or we're migrating incorrectly we should figure it out, but this will cover up another problem that i'd rather get to the root of unless you think there's a reason it would ever be undefined